src/Components/NavBar/Menus/MobileLink.tsx

```diff
@@ -12,10 +15,23 @@ interface MobileLinkProps {
 export const MobileLink: React.FC<MobileLinkProps> = ({
   href,
   children,
+  contextModule,
   ...props
 }) => {
   const [isPressed, setPressed] = useState(false)
   const bg = isPressed ? "black5" : "white100"
+  const { trackEvent } = useTracking()
+  const text = children.toString()
+
+  const handleClickTracking = (linkHref: string) => {
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.Click,
+      context_module: contextModule,
```

Comment:
This is great! Nice refactor.

One last question. It _looks like_ the vast majority of the `contextModule`s that are passed in have the same value of `AnalyticsSchema.ContextModule.Header`. If that's the case, can we give this a default value and make the prop optional?